flexmeasures/cli/data_add.py

```diff
@@ -84,6 +86,27 @@ def fm_add_data():
     """FlexMeasures: Add data."""
 
 
+@fm_add_data.command("data-gen-sources")
```

Comment:
I'd like to see this functionality aligned or even integrated with `flexmeasures add source`. For example, `flexmeasures add source --type reporters` could be made to do what `flexmeasures add data-gen-sources` currently does.

Or, if it makes sense to keep the `add source` command reserved for adding a single source only (this could be a discussion), the command could become `flexmeasures add sources --type reporters`.

Likewise, to register both reporters and schedulers:
```
flexmeasures add sources --type reporters --type schedulers
```
Or to register all data generators:
```
flexmeasures add sources --type data-generators
```
or even just:
```
flexmeasures add sources
```

My preference for now—because this is still a somewhat experimental feature—would be to stick with the separate command `add sources` for this functionality.

Reply:
Good idea!